### PR TITLE
Add minutes for meeting of 04/24/2015

### PR DESCRIPTION
**Pull request by mhdawson**
Add minutes for meeting of 04/24/2015

### CODE REVIEW

File: doc/about/core-team/meetings/2015-04-24/minutes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+If the two code bases are merged not including ICU would be a forced break.  Current
+understanding is that merged code base would be everything from both code bases.  
+If instead individual pieces are not going to be included then likely convergence working 
+group would need to build list to cover these pieces and work through an appropriate
+approach to deal with each one. 
+
+In advance, we should start looking to see if we can make the inclusion of ICU more
+modular in a non breaking way.  Action: Michael to setup meeting with Steven/James
+to better understand depencies between startup and V8 with respect to ICU.
```
Comment (geek) [inline doc/about/core-team/meetings/2015-04-24/minutes.md]: typo: `dependencies`

File: doc/about/core-team/meetings/2015-04-24/minutes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+match this descripton (James), finalize release notes (Julien with input from all) 
+and then do one final review before release (all).   Important that we add
+tests to make sure merge with 0.12.X can be easily validated.
+
+#### v0.12.3
+
+Same discussion of removal of RC4 to address Bar Mitzva attack.  Just need
+to do commits in a way that facilitates merge from 0.10.X later
+
+Julien: looking at completeing V8 lateral upgrade.  Will be v0.12.3 unless
```
Comment (geek) [inline doc/about/core-team/meetings/2015-04-24/minutes.md]: typo: `completing`

File: doc/about/core-team/meetings/2015-04-24/minutes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+removal of RC4 is ready before it is.
+
+### Internationalization
+
+James: io.js looking at Globalize as an alternative, sounds like because they would like
```
Comment (geek) [inline doc/about/core-team/meetings/2015-04-24/minutes.md]: reword: `, because they would like...`

**Comment by mhdawson** [comment]
3 comments so far addressed.

**Comment by misterdjules** [comment]
@mhdawson Thank you!

`doc/about/core-team/meetings/index.md` needs to be updated with a new entry for these minutes (including a link to it). You can test that the website is generated properly by running `make test`.

**Comment by mhdawson** [comment]
I seem to get an error when running make tests. IT complains about 

hecking for doc/nodejs-advisory-board/governance-proposal/TSC-Project-Lifecycle.json
using template doc/website.html
/home/mhdawson/node-website-pull/node-website/tools/doc/generate.js:118
        if (er) throw er;
                      ^
Error: Inappropriate heading level
{"type":"heading","depth":2,"text":"**_DRAFT for Public Comment**_"}
    at /home/mhdawson/node-website-pull/node-website/tools/doc/html.js:163:17
    at Array.forEach (native)
    at buildToc (/home/mhdawson/node-website-pull/node-website/tools/doc/html.js:160:9)
    at render (/home/mhdawson/node-website-pull/node-website/tools/doc/html.js:48:3)
    at /home/mhdawson/node-website-pull/node-website/tools/doc/html.js:34:5
    at fs.js:334:14
    at FSReqWrap.oncomplete (fs.js:95:15)
make: **\* [out/doc/nodejs-advisory-board/governance-proposal/TSC-Project-Lifecycle.html] Error 1

**Comment by mhdawson** [comment]
PR to update index https://github.com/joyent/node-website/pull/109

**Comment by mhdawson** [comment]
PR to fix markdown errors in advisory board files:
https://github.com/joyent/nodejs-advisory-board/pull/40

**Comment by mhdawson** [comment]
I now get as far as:

open http://localhost:8080
Couldn't get a file descriptor referring to the console
make: **\* [test] Error 1

I assume that's because I would have to setup a web server ?

**Comment by misterdjules** [comment]
@mhdawson The web server [is started after open is started](https://github.com/joyent/node-website/blob/master/Makefile#L99). It works on OSX, but I haven't tested it on other platforms, so there may be a bug here.

**Pull request merged**